src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
+  .${PLACEHOLDER_CLASS}::before {
+    position: absolute;
+    content: '${text}';
+    cursor: text;
+  };
+</style>`;
+
+    span.className = PLACEHOLDER_CLASS;
```

Comment:
What's the point of the class? Do we use it anywhere? Oh, just if the user wants to hook into it? I don't think we should expose that, tbh.